common/oatbox/service/ServiceFactoryInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Declare statements should be placed below the file-level docbloc, BTW.

https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements